src/homework4/7_looking_back.py

```diff
@@ -0,0 +1,11 @@
+# Даны два натуральных числа. Вычислите их наибольший общий делитель
+# при помощи алгоритма Евклида (мы не знаем функции и рекурсию).
+
+num1 = int(input("введите первое число:"))
+num2 = int(input("введите второе число:"))
+while num1 != 0 and num2 != 0:
+    if num1 > num2:
+        num1 = num1 % num2
+    else:
+        num2 = num2 % num1
+print(num1 + num2)
```

Comment:
algorithm suppose result not as a sum (it's a little hack from your side not to miss where it is.